migrations/2024-08-28-124527_add-repo-submissions-table/down.sql

```diff
@@ -0,0 +1,4 @@
+-- This file should undo anything in `up.sql`
+DROP TABLE submissions;
+DROP TABLE repositories;
```

Comment:
```suggestion
DROP TABLE repositories;
DROP TABLE submissions;
```

Reply:
The order should be switched since `submissions` has a dependency on `repositories`.

Reply:
Good catch!

Reply:
addressed here -> 1deec9aca94e30d3d33a26f8cbe5207e3dd99d8d